src/docs/product/performance/resources.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Resources
+sidebar_order: 50
+description: "Learn more about browser resource performance monitoring, which allows you to see debug the performance of loading JavaScript and CSS on your frontend."
+---
+
+If you have performance monitoring enabled for your frontend, you can see how your browser resources are performing in Sentry.
+
+Starting with the [**Resources** page](#resources-page), you get a high-level overview of how your resources are doing. From there, you can drill into a specific resource's [**Resource Summary** page](#resource-summary-page) and then investigate sample events from the [Sample List](#sample-list) to better understand the context of its performance in a specific page.
+
+<!-- TODO add gif -->
+
+## Prerequisites and Limitations
+
+The resource pages are only available for frontend JavaScript projects with performance monitoring enabled. Currently only JavaScript and CSS resources are supported.
```

Comment:
Technically in the current state this is `resources initiated by css` as we are still relying on initatorTypes for span ops. @AbhiPrasad what do you think?

Reply:
Yeah we can explicitly link to the MDN docs as well, let's change it to resources initiated by css/css `url` function` and script tags.

Reply:
@AbhiPrasad thoughts on this? Also after this minor edit, i think docs should be good to merge!

```suggestion
The resource pages are only available for frontend JavaScript projects with performance monitoring enabled. Currently only JavaScript, CSS, fonts and certain resources initiated by css such as fonts are supported.
```

Reply:
Based on the recent file extension change, We technically don't support all resources initiated by css such as images, which is why I added "certain resources"